swagger.json

```diff
@@ -1121,6 +1121,77 @@
             }
           }
         },
+        "/fleet/drivers/{driver_id}/hos_daily_log": {
+            "patch": {
+              "consumes": [
+                "application/json"
+              ],
+              "description": "Modify details, such as trailer number or shipping ID, for a driver's daily log",
+              "operationId": "edit_fleet_drivers_hos_daily_log_details",
+              "parameters": [
+                {
+                  "$ref": "#/parameters/accessTokenParam"
+                },
+                {
+                  "description": "ID (integer) of the driver with HOS logs.",
+                  "in": "path",
+                  "name": "driver_id",
+                  "required": true,
+                  "type": "string"
+                },
+                {
+                  "in": "body",
+                  "name": "editHosDailyLogParams",
+                  "required": true,
+                  "schema": {
+                    "properties": {
+                      "timestamp": {
```

Comment:
In most of our other APIs we use the key `timeMs`. Do we want to do the same here?